src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -120,6 +120,9 @@ export class TechRecordSummaryComponent implements OnInit {
         return this.getHgvTemplates();
       case VehicleTypes.TRL:
         return this.getTrlTemplates();
+      // TODO: Double check this
+      default:
+        return this.getPsvTemplates();
```

Comment:
There are no light vehicle template functions in the tech records at the moment, so I've just had them default to the PSV functions

Reply:
Proper pedantic but can you make the TODO slightly clearer

Reply:
Oh this is a draft, so it's fine

Reply:
They've been rewritten now